packages/@aws-cdk/core/test/test.app.ts

```diff
@@ -245,65 +246,88 @@ export = {
     test.done();
   },
 
-  'runtime library versions disabled'(test: Test) {
-    const context: any = {};
-    context[cxapi.DISABLE_VERSION_REPORTING] = true;
-
-    const assembly = withApp(context, app => {
-      const stack = new Stack(app, 'stack1');
-      new CfnResource(stack, 'MyResource', { type: 'Resource::Type' });
+  /**
+   * Runtime library versions are now synthesized into the Stack templates directly
+   *
+   * The are not emitted into Cloud Assembly metadata anymore
+   */
+  'runtime library versions are not emitted in asm anymore'(test: Test) {
+    withCliVersion(() => {
+      const context: any = {};
+
+      const assembly = withApp(context, app => {
+        const stack = new Stack(app, 'stack1');
+        new CfnResource(stack, 'MyResource', { type: 'Resource::Type' });
+      });
+
+      test.deepEqual(assembly.runtime, { libraries: {} });
     });
-
-    test.deepEqual(assembly.runtime, { libraries: {} });
     test.done();
   },
 
   'runtime library versions'(test: Test) {
```

Comment:
`~` It feels like this test is slightly redundant with the `'version reporting includes only @aws-cdk, aws-cdk and jsii libraries'` test.